packages/components/src/vue/vuex-modules/data/lexis.js

```diff
@@ -190,7 +210,7 @@ export default class Lexis extends Module {
     this._treebankDataItem = treebankDataItem
     let annotatedHomonyms
 
-    if (this._treebankReady && this._treebankDataItem) {
+    if (this._treebankReady && this._treebankDataItem && !this.isFailedProvider(this._treebankDataItem)) {
```

Comment:
under what circumstances would _treebankReady be true AND isFailedProvided also be true?

Reply:
I think you're right, they are exact opposites now. `_treebankReady` has an additional functionality before that is now not used now. I think there is no reason for `_treebankReady` any more. I will update the code to reflect that.